app/models/form_profile.rb

```diff
@@ -220,7 +220,8 @@ def initialize_identity_information
     )
   end
 
-  def convert_vets360_address(address)
+  def vet360_mailing_address_hash
```

Comment:
what about `def vet360_mailing_address_hash(include_street3 = false)` so that it's more clear to others who come along later that address line3 is even a thing?

Reply:
that hash is used as arguments to create a FormAddress object (via a FormContactInfo object).
so it could be added to `vet360_mailing_address_hash`, but would be ignored unless i add street3 as an attribute to a FormAddress.
Changing FormAddress would change the prefills for a handful of prefills which send the entire FormAddress object to the FE (suddenly all of those prefills would have an extra field). It's hard to say whether or not that would be a problem (could cause schemas to fail).
I decided to err on the side of caution and not change FormAddress.